.circleci/config.yml

```diff
@@ -113,6 +114,7 @@ jobs:
       - run:
           name: Install trivy
           command: |
+            apt-get install -y sudo
             curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/master/contrib/install.sh | sudo sh -s -- -b /usr/local/bin
```

Comment:
Are you already running as `root`? If so, can you run this without `sudo`?

Reply:
I will try running without `sudo`